Formula/i/idsgrep.rb

```diff
@@ -0,0 +1,38 @@
+class Idsgrep < Formula
+  desc "Grep for Extended Ideographic Description Sequences"
+  homepage "https://tsukurimashou.org/idsgrep.php.en"
+  url "https://tsukurimashou.org/files/idsgrep-0.6.tar.gz"
+  sha256 "2c07029bab12d9ceefddf447ce4213535b68d020b093a593190c2afa8a577c7c"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url :homepage
+    regex(/href=.*idsgrep[._-]v?(\d+(?:\.\d+)+)\.t*/i)
+  end
+
+  depends_on "cmake" => :build
+  depends_on "pcre"
```

Comment:
any upstream issue to support pcre2?

Reply:
> ```
>   # From the PCRE homepage:
>   # "The older, but still widely deployed PCRE library, originally released in
>   # 1997, is at version 8.45. This version of PCRE is now at end of life, and
>   # is no longer being actively maintained. Version 8.45 is expected to be the
>   # final release of the older PCRE library, and new projects should use PCRE2
>   # instead."
> ```


but pcre is EOL now

Reply:
Added a compile option to not use pcre.

Reply:
pcre is not deprecated in the homebrew yet, but I was mostly just curious if we can ask upstream to switch off using pcre. does that make better sense?